tests/config.test.ts

```diff
@@ -3,7 +3,7 @@ import Romcal from '../lib';
 const { RomcalConfig, LiturgicalDayConfig } = Romcal;
 
 describe('getConfig()', () => {
-  test('should get general config if country doesnt have default configurations', async () => {
+  test('should get general config if country doesn't have default configurations', async () => {
```

Comment:
The `'` need to be escaped here, otherwise it creates a syntax error in the codebase. 

Or better, to keep it friendly to read (and consistent test codebase): 
```suggestion
  test('should get general config if country does not have default configurations', async () => {
```

Reply:
Yeah, I missed that, thanks, @emagnier! 🙏